quaddtype/release_tracker.md

```diff
@@ -0,0 +1,93 @@
+# Plan for `numpy-quaddtype` v1.0.0
+
+| ufunc name    | Added |
```

Comment:
Could you add a column for which ufuncs have tests against float and np.float64 that ensure the correct handling of edge cases (NaN, inf, -0.0, points in [0, 2pi], ...)? I'd like to ensure that by publication we can guarantee that all ufuncs comply with the numpy requirements (see e.g. my issue about mod being incorrect right now)

Reply:
I see, sure will add the corresponding column as well